fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/digitalcash/DigitalCashMain.java

```diff
@@ -26,6 +31,77 @@ protected void onCreate(Bundle savedInstanceState) {
     mViewModel = obtainViewModel(this);
 
     setupNavigationBar();
+    // Subscribe to "open home"
+    mViewModel
+        .getOpenHomeEvent()
+        .observe(
+            this,
+            booleanEvent -> {
+              Boolean event = booleanEvent.getContentIfNotHandled();
+              if (event != null) {
+                setupFragment(R.id.fragment_digital_cash_home);
+              }
+            });
+
+    // Subscribe to "open history"
+    mViewModel
+        .getOpenHistoryEvent()
+        .observe(
+            this,
+            booleanEvent -> {
+              Boolean event = booleanEvent.getContentIfNotHandled();
+              if (event != null) {
+                setupFragment(R.id.fragment_digital_cash_history);
+              }
+            });
+
+    // Subscribe to "open send"
+    mViewModel
+        .getOpenSendEvent()
+        .observe(
+            this,
+            booleanEvent -> {
+              Boolean event = booleanEvent.getContentIfNotHandled();
+              if (event != null) {
```

Comment:
Maybe add some LOG statement, useful to debug if anyone touche your code while subscribing

Reply:
Comment was take into account (added the logs)